online-hacknight/index.html

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Code for Münster
+page_class: online-hacknight
+---
+<div class="wrapper">
+  <div id="online-hacknight">
+    <div class="text" onload="window.location = 'https://jitsi.osna.social/code-for-muenster'">
```

Comment:
Hier und in Zeile 14 und 16 sollte meiner Meinung nach https://hackmd.okfn.de/codeformuenster-online-hacknights verlinkt sein

Reply:
Dann hast du aber wieder zwei Klicks und die Webseite ist doch (fast) genauso schnell zu bearbeiten wie das Hackpad. Den Text können wir ja auch noch hier einbauen, falls jemand nicht sofort weitergeleitet wird.